plugins/importexport/native/filter/NativeXmlSubmissionFilter.php

```diff
@@ -104,6 +106,9 @@ public function handleElement($node)
         $submission = Repo::submission()->get($submission->getId());
 
         $deployment->addImportedRootEntity(Application::ASSOC_TYPE_SUBMISSION, $submission);
+        //add Empty review stage for reviewed  submissions
```

Comment:
If we would rename the new function either to createReviewRound or handleSubmissionReviewRound, maybe no comment here would be needed. 
Else maybe to have: "create a review round for submissions in review" ?

Reply:
got rid of this obsolete comment after above changes